doc/storage/appendix/sra.rst

```diff
@@ -0,0 +1,919 @@
+
+  
+  .. figure:: /figure/protected.*
+   :name: fig-protected
+
+   Trust boundaries in the Deployment Model `DM.PROTECTED`
+  
+
+:deployment-model:`EXPOSED`
+
+  The :term:`PRoT` partition does not have sole access to the area of non-volatile storage, thus the storage can be read or written by another partition or by other means. This means that the driver code, or the storage interface or the Storage Medium resides outside the :term:`PRoT` and is accessible to other partitions or by other means, as shown in  as shown in :numref:`fig-exposed`. Therefore, attackers can bypass the Storage Service. 
+  
+  The Storage Service is the arbitrator of access from different applications and manages those data accesses (write, update and deletion). Therefore, the Storage Service is responsible for the `SG.CONFIDENTIALITY` goal requirements of each caller.
+  
+  The Storage Service cannot prevent other partitions or other means from reading or writing the storage. Therefore, the Storage Service must support the `SG.CONFIDENTIALITY`, `SG.INTEGRITY` and `SG.CURRENCY` goals. 
+  
+  An example of this deployment model is the use of a file system on a flash chip. 
+
+
+  .. figure:: /figure/exposed.*
+   :name: fig-exposed
+
+   Trust boundaries in the Deployment Model `DM.EXPOSED`
+
+ 
+:deployment-model:`AUTHORISED`
+
+  There is a separate isolated partition containing an area of non-volatile storage that can only be accessed in response to an authenticated command and from which all replies include a means for verification of the response, as shown in :numref:`fig-authorised`. The isolation guarantees that there is no access to the storage other than by using the authentication mechanism. 
+  
+  Therefore, the authentication mechanism must be protected.  In other words, the means of generating the authenticated command and verifying the response must be available only to the Storage Service. This implies that there are confidentiality and integrity goals on any secrets used for this purpose.
+  
+  The authorisation and verification mechanism provided by the storage controls access to data (reads, writes and modification). This supports the `SG.INTEGRITY` and `SG.CURRENCY` goals.
+  
+  However, the communication between the Storage Service and the Storage Medium is observable by other partitions and any other means as any data sent in plain text can be observed. Therefore, the Storage Service must support `SG.CONFIDENTIALITY`.  For cryptographic based solutions, this implies that there are confidentiality and integrity goals on any secrets used in cryptographic implementation.
+  
+  An example of this deployment model is the use of an RPMB memory block. 
+
+
+  .. figure:: /figure/authorised.*
+   :name: fig-authorised
+
+   Trust boundaries in the Deployment Model `DM.AUTHORISED`
+
+  
+:deployment-model:`SECURE_CHANNEL`

```

Comment:
A minor issue. The length of this deployment model tag/name slightly disturbs the layout of the risk reporting in the Threats. If this could be a shorter tag, it would improve the visual clarity of those sections. Is 'SECURE_LINK' a reasonable alternative?